src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpAgoraDAO.scala

```diff
@@ -38,6 +40,18 @@ class HttpAgoraDAO(config: FireCloudConfig.Agora.type)(implicit val system: Acto
     authedRequestToObject[List[EntityAccessControlAgora]]( Post(getMultiEntityPermissionUrl(entityType), entities) )
   }
 
+  override def batchCreatePermissions(inputs: List[EntityAccessControlAgora])(implicit userInfo: UserInfo): Future[List[EntityAccessControlAgora]] = {
```

Comment:
new methods since we're no longer firing off direct (but non-passthrough) requests from `AgoraPermissionActor` anymore